### DISCUSSION TITLE
What highlight group is this?

### DISCUSSION THREAD

**Discussion by katawful** [nvim-neorg/neorg#231] [Q&A] [answered]
![2021-12-11~12:05:28](https://user-images.githubusercontent.com/45222045/145685230-2837c877-b8a1-4c37-ab35-4a9b94e3ff17.png)
I like having colors stand for these virtual texts, but the purple and green texts aren't using the same background colors. Not sure what groups they use

**Comment by vhyrro** [comment, marked as answer]
maan, why do I keep missing these discussions. Sorry!

In case you didn't figure it out yourself yet, the purple you see is `TSField`, and the total (the "of 1" string) is `NeorgTodoItem1Done`, which by default links to `TSString`.
The default highlight for the text itself is `DiagnosticVirtualTextHint`.

The actual format for the base completion string can be found [here](https://github.com/nvim-neorg/neorg/blob/main/lua/neorg/modules/core/norg/concealer/module.lua#L112-L130). Hope that helps!

**Reply by katawful** [reply]
Thanks!

**Reply by unblevable** [reply]
Hi, any update to which highlight group this would apply to with the `core.highlights` revamp
